packages/components/src/AudioPlayer/index.tsx

```diff
@@ -132,7 +132,7 @@ export const AudioPlayer: React.FC<AudioPlayerProps> = ({ src, title, skipDurati
         <Caption6>{durationString}</Caption6>
       </div>
 
-      <ButtonBar className={styles.buttonBar} stackAtBreakpoint={0}>
+      <ButtonBar className={styles.buttonBar} stackAtBreakpoint={0} disableAutoAlignment={true}>
```

Comment:
```suggestion
      <ButtonBar className={styles.buttonBar} stackAtBreakpoint={0} disableAutoAlignment>
```

Value must be omitted for boolean attributes: [eslint react/jsx-boolean-value](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/jsx-boolean-value.md)